### PR TITLE
Commented out the argument as Pods crashing due to argument  ["--enab…

### PR DESCRIPTION
**Pull request by MATRIX4284**
…le-session"]

Commented out the argument in the yaml as pods are creashing due to the below argument:
args: ["--enable-session"] this argument is not valid causing the ppod to crashback

### CODE REVIEW

**Review by wangkuiyi — COMMENTED** [review]
If this argument is no longer required,how about removing it?

**Review by wangkuiyi — APPROVED** [review]

**Review by wangkuiyi — CHANGES_REQUESTED** [review]
Thanks for the PR! I am holding the merge of this PR because we need @typhoonzero and @Yancey1989 to confirm that sessions could be disabled.

**Comment by Yancey0623** [comment]
The argument `--enable-session` has been removed because of `sqlflowserver` enable session by default.

File: doc/k8s/sqlflow-server.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ spec:
         name: sqlflow-server
         imagePullPolicy: Always
         command: ["sqlflowserver"]
-        args: ["--enable-session"]
```
Comment (Yancey0623) [inline doc/k8s/sqlflow-server.yaml:22]: Please remove this line as the comment: https://github.com/sql-machine-learning/sqlflow/pull/1632#issuecomment-571648841

Comment (wangkuiyi) [reply doc/k8s/sqlflow-server.yaml:22]: Thanks @Yancey1989 .  I am doing this quick change and merge.

**Review by wangkuiyi — APPROVED** [review]

**Pull request merged**